<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frequency</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="5,0,5,1,1">
   <item>
    <widget class="QGroupBox" name="verticalGroupBox">
     <property name="title">
      <string>Enter sampling frequency (Hz)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="edit_freq">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Enter sampling frequency in Hz</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>OR</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="">
     <property name="title">
      <string>Choose recent sampling frequency (Hz)</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Recent values:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combo_recent"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_use_recent">
        <property name="text">
         <string>Use</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Tip: Press Ctrl+U or Cmd+U to trigger the 'Use' button.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
